llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -345,7 +349,7 @@ static VPRegionBlock *createReplicateRegion(VPReplicateRecipe *PredRecipe,
   auto *RecipeWithoutMask = new VPReplicateRecipe(
       PredRecipe->getUnderlyingInstr(),
       make_range(PredRecipe->op_begin(), std::prev(PredRecipe->op_end())),
-      PredRecipe->isUniform());
+      PredRecipe->isUniform(), /*Mask*/ nullptr, *PredRecipe);
```

Comment:
```suggestion
      PredRecipe->isUniform(), nullptr /*Mask*/, *PredRecipe);
```
and similarly below?